fe/fe-core/src/main/java/com/starrocks/catalog/HiveTable.java

```diff
@@ -297,7 +291,8 @@ public long getPartitionStatsRowCount(List<PartitionKey> partitions) {
         return HiveMetaStoreTableUtils.getPartitionStatsRowCount(hmsTableInfo, partitions);
     }
 
-    private void validate(Map<String, String> properties) throws DdlException {
+    private void validate(Map<String, String> properties, org.apache.hadoop.hive.metastore.api.Table hmsTable)
```

Comment:
how about naming this as hiveTable

Reply:
done, but there is a var named hiveTable in '340'. I convert it to hmsTable.